src/native/corehost/hostmisc/pal.windows.cpp

```diff
@@ -648,8 +648,8 @@ bool pal::realpath(string_t* path, bool skip_error_logging)
     if (LongFile::IsNormalized(*path))
     {
         WIN32_FILE_ATTRIBUTE_DATA data;
-        if (path->empty() // An empty path doesn't exist
-            || GetFileAttributesExW(path->c_str(), GetFileExInfoStandard, &data) != 0)
+        if (!path->empty() // An empty path doesn't exist
```

Comment:
Can we just check for empty / return false immediately? Seems odd and unnecessary to go through `LongFile::IsNormalized` and then rely on `GetFullPathNameW` to fail for an empty path.